Zapper-fi/studio
# feat(rigoblock): add uni v3 positions

## Description
Added Rigoblock contract position fetcher for Uniswap V3 liquidity positions owned by Rigoblock pools. The contract positions are with the same group id as the token fetcher. This PR also adds base currency balances (i.e. ETH for Ethereum and so on).



## Checklist


## How to test?
run locally and check the returned values from http://localhost:5001/apps/rigoblock/positions?groupIds[]=pool&network=ethereum
